blocks/identity-block/intl.json

```diff
@@ -240,5 +240,15 @@
 		"no": "Sign up for an account.",
 		"pt": "Sign up for an account.",
 		"sv": "Sign up for an account."
-	}
+	},
+	"identity-block.account-information": {
```

Comment:
@JackHowa - These translations need to be added to each of the `locale/{name}.json` files too

Reply:
updated https://github.com/WPMedia/fusion-news-theme-blocks/pull/1180